JobAnalyzer.py

```diff
@@ -44,12 +51,14 @@
 logger.setLevel(logging.INFO)
 
 class JobCost:
-    def __init__(self, job: SchedulerJobInfo, spot: bool, instance_family: str, instance_type: str, rate: float):
+    def __init__(self, job: SchedulerJobInfo, spot: bool, instance_family: str, instance_type: str, rate: float, compute_sp_rate: float, ec2_sp_rate: float):
```

Comment:
Initially I thought that this would be useful for the analysis.
However, with the addition of the xls spreadsheet I don't think that this is actually necessary.
Not sure if it's worth taking it back out.

Reply:
I'm going to take these back out.